### PR TITLE
make sample code in README executable (partially)

### PR DESCRIPTION
**Pull request by nov**
Great gem!

I fixed 2 tiny bugs which made your sample code causes errors.
I'm still getting an error at `Keynote::ArrayMethods#<<`, but I'm not sure it's because my MacOS is Yosemite or not. (`eval_script` returns `nil`)

Looking forward the v1.0.0 release ;-)

### CODE REVIEW

File: lib/keynote/document.rb [hunk 1]
```diff
@@ -145,7 +146,7 @@ def self.create(arguments = {})
         JSON.stringify(doc.properties());
       APPLE
 
-      self.new(symbolize_keys(result).merge(theme: theme, width: width, height: height))
+      self.new(symbolize_keys(result).merge(theme: theme, width: width, height: height, file_path: file_path))
```
Comment (katsuma) [inline lib/keynote/document.rb]: __[rubocop]__
* Redundant `self` detected.
* Line is too long. [110/80]

**Comment by katsuma** [comment]
Sorry for my late reply. Thank your for your PR. :bow:

**Comment by katsuma** [comment]
@nov 

> I'm still getting an error at Keynote::ArrayMethods#<<, but I'm not sure it's because my MacOS is Yosemite or not. (eval_script returns nil)

I found I made a mistake `Yosemite` for `Mavericks` :sweat_drops:  ([and I fixed it.](https://github.com/katsuma/keynote-client/pull/8))
So your environment looks good.

So when you have time, can you try this?

1. Check out the latest `keynote-client` code.
2. Comment out line16 of `lib/util.rb` (`file.delete`)
3. Try `bundle exec pry` and sample code, for example. `require 'keynote-client'; include Keynote; ... `
4. Open a new terminal window, go to `/tmp` and you'll find `osascript#{random-chars}.js`
5. Try `osascript -l JavaScript osascript#{random-chars}.js`

This may help us to understand what happens.

**Comment by nov** [comment]
I could made a keynote file by this code.
https://gist.github.com/nov/fd3c7f88656b920c5380

I'm still getting a JS syntax error because the sample code ends up a newline in the JS string :p
```
nov.matake@tovan tmp$ osascript -l JavaScript /var/folders/0n/bnn0n_xs61xcnrqc_86mppnr2nfpm2/T/osascript20151207-21086-ajci1y.js
/var/folders/0n/bnn0n_xs61xcnrqc_86mppnr2nfpm2/T/osascript20151207-21086-ajci1y.js:162:219: script error: Error on line 4: SyntaxError: Unexpected EOF (0)
nov.matake@tovan tmp$ cat /var/folders/0n/bnn0n_xs61xcnrqc_86mppnr2nfpm2/T/osascript20151207-21086-ajci1y.js
        var Keynote = Application("Keynote")
        var doc = Keynote.documents.byId("C66760E6-9EC5-4758-9709-1D3594C099EC")
        var slide = doc.slides()[1]
        slide.defaultBodyItem.objectText = "This is a pen
Is this a pen?"
        JSON.stringify({ result: true })
```

Anyway, great work!

**Comment by katsuma** [comment]
@nov 
Maybe master code is broken now... :cold_sweat: 
I got the same error, too.

Can you try gem version when you have time?

```sh
gem install keynote-client
pry
```
```
require 'keynote-client'
include Keynote
...
```

I could append a new slide to my keynote document. (`doc.slides << slide`)

**Comment by katsuma** [comment]
Thank your for your quick reply.
I'll check the code :smile:

**Comment by katsuma** [comment]
@nov 
Maybe you got this error when you try to re-set `Slide#body=` with string including new-line character??
(Your log seems calling `slide.body = "This is a pen\nIs this a pen?"`)

I confirmed `Slide#title=` and `Slide#body=` have issue to fail generating JavaScript code.
And I released a new version `v0.1.2` fixing this issue.

**Comment by nov** [comment]
The error occurs [here](https://gist.github.com/nov/fd3c7f88656b920c5380#file-keynote-client-sample-rb-L24).
I think it's not "re-set" but "set".

Your commit https://github.com/katsuma/keynote-client/commit/a1e2c74ce6eb7dc05919c848437370021510b75f will fix my case, but isn't it better to JSONize the string like this?
https://gist.github.com/nov/4c3ba82bede0de88c100

**Pull request merged**